File: src/mtvap_1266.rs
pub fn start(points: Vec<Vec<i32>>) -> i32 {
    if points.len() == 1 {
        return 0;
    }

    let mut res: i32 = 0;
    for i in 1..points.len() {
        let x = (points[i][0] - points[i - 1][0]).abs();
        let y = (points[i][1] - points[i - 1][1]).abs();
        res += x.max(y)
    }

    return res;
}

File: mtvap_1266.js
/**
 * @param {number[][]} points
 * @return {number}
 */
const minTimeToVisitAllPoints = function (points) {
  if (points.length == 1) {
    return 0;
  }

  let res = 0;
  for (let i = 1; i < points.length; i++) {
    // console.log('1', Math.abs(points[i][0] - points[i - 1][0]));
    // console.log('2', Math.abs(points[i][1] - points[i - 1][1]));
    res += Math.max(
      Math.abs(points[i][0] - points[i - 1][0]),
      Math.abs(points[i][1] - points[i - 1][1])
    );

    // console.log('res', res);
  }
  return res;
};

// console.log('test1', test1);
// console.log('test2', test2);
